app/models/requests/requestable.rb

```diff
@@ -281,6 +281,10 @@ def cul_music?
       item[:collection_code].present? && item[:collection_code] == 'MR'
     end
 
+    def resource_shared?
```

Comment:
@mzelesky will any items that find themselves in various resource sharing workflows always fine themselves in the RES_SHARE library?

Reply:
Do you mean our own items that are loaned to other institutions?